### ISSUE TITLE
Second startup of all-in-one takes 100 seconds to finish starting

### ISSUE THREAD

**Issue by liggitt** [openshift/origin#899]
On first startup, a node is auto created

On second startup, the node exists, and messages about the node already existing are printed 10 times, 10 seconds apart, until it gives up and continues.
```
start.go, 550: kmaster.RunMinionController()
   kubernetes/master.go, 143: minionController.Run(10*time.Second, 10)
      nodecontroller.go, 122: _, err := s.kubeClient.Nodes().Create(&node)
```

Looks like nodecontroller.go tolerated an "already exists" error until the latest upstream bump

**Comment by liggitt** [comment]
@smarterclayton

**Comment by liggitt** [comment]
Sample output:
```
I0204 23:04:15.733976   22975 master.go:111] Started Kubernetes Endpoint Controller
2015/02/04 23:04:15 http: TLS handshake error from 127.0.0.1:55292: EOF
E0204 23:04:16.000618   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:04:26.083406   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:04:36.124428   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:04:46.155880   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:04:56.176282   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:06.201227   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:16.221672   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:26.246466   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:36.275427   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:46.295528   22975 nodecontroller.go:127] Error registrying node jliggitt-laptop, retrying: minion "jliggitt-laptop" already exists
E0204 23:05:56.295805   22975 nodecontroller.go:93] Error registrying node list &{TypeMeta:{Kind: APIVersion:} ListMeta:{SelfLink: ResourceVersion:} Items:[{TypeMeta:{Kind: APIVersion:} ObjectMeta:{Name:jliggitt-laptop GenerateName: Namespace: SelfLink: UID: ResourceVersion: CreationTimestamp:0001-01-01 00:00:00 +0000 UTC Labels:map[] Annotations:map[]} Spec:{Capacity:map[cpu:{Amount:1.000 Format:DecimalSI} memory:{Amount:3221225472 Format:BinarySI}]} Status:{HostIP:127.0.0.1 Phase: Conditions:[{Kind:Ready Status:None LastTransitionTime:0001-01-01 00:00:00 +0000 UTC Reason: Message:}]}}]}: unable to register all nodes.
I0204 23:05:56.296147   22975 master.go:145] Started Kubernetes Minion Controller
I0204 23:05:56.296894   22975 master.go:595] OpenShift UI listening at https://0.0.0.0:8444
I0204 23:05:56.540056   22975 master.go:606] OpenShift UI available at https://192.168.1.123:8444
```

**Comment by smarterclayton** [comment]
We need to fix the loop thing that handles already exist.  Not sure why that didn't get fixed upstream.
